cmake/Modules/FindLibEv.cmake

```diff
@@ -0,0 +1,44 @@
+# EOS - The CERN Disk Storage System
+# Copyright (C) 2015 CERN/Switzerland
+# This program is free software: you can redistribute it and/or modify it under
```

Comment:
Could you please ask them if they can relicense this trivial code as BSD/public domain? If it is copyright-able at all: The license header is as long as the code. Where did you get it from?

Reply:
There are a few available online. I found some GPLv2, GPLv3, Apache, MIT, but no BSD. The [original file](https://gitlab.cern.ch/dss/eos/blob/ea596793395a846038e5e02326d804a6b9cda8d8/cmake/Findlibev.cmake) had no license header, I added the project's license header.